test/integration/testdata/rbac-beta.yaml.tmpl

```diff
@@ -12,7 +12,10 @@ rules:
   resources: ["istiokinds"]
   verbs: ["get", "list", "watch"]
 - apiGroups: ["apiextensions.k8s.io"]
-  resources: ["customresourcedefinitions"]
+  resources: ["*"]
```

Comment:
You asked me to make this change in istio/istio to be future proof, made the same change in pilot.